test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -843,7 +843,7 @@ def test_oss_compliance(image):
                 try:
                     if not os.path.isdir(local_file_path):
                         context.run(
-                            f"git clone {url.rstrip()} {local_file_path}")
+                            f"git clone {url.rstrip()} {local_file_path}", hide=True)
```

Comment:
Is there a specific reason to add this? It is a good addition though, just want to understand if it helps us resolve any failure.

Reply:
Yes, we added that so that we can see the complete error trace when failure occurs.